introduction.md

```diff
@@ -0,0 +1 @@
+I am new in the dojo. Practicing the forks. New rules \n *Unicorns allowed \n *Pijamas are compulsory
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.